src/routes/parkRoutes.js

```diff
@@ -32,6 +32,7 @@ router.route('/initiateparksdb')
       headers: {
         'Accept': 'application/json',
         'Content-Type': 'application/json',
+        // Might be safer to fetch your authorization key from an env variable - Harold
```

Comment:
we will remove this route entirely.  It was being used to get our initial park data.  No longer needed.